modules/engage-theodul-plugin-controls/pom.xml

```diff
@@ -151,12 +151,13 @@
           </instructions>
         </configuration>
       </plugin>
+<!--
```

Comment:
If this can go, remove it?
Commenting this out just creates dead code.

Reply:
Bah, I meant to go back and do that.  Thanks for the reminder.